packages/orchestrator-ui-components/src/rtk/api.ts

```diff
@@ -48,6 +53,24 @@ export const handlePromiseErrorWithCallback = <T>(
     });
 };
 
+export const catchErrorResponse: ResponseHandler = async (
+    response: Response,
+) => {
+    if (
+        response.status < HttpStatus.Ok ||
+        response.status >= HttpStatus.MultipleChoices
+    ) {
+        console.error(response.status, response.body);
+    } else if (
+        response.status === HttpStatus.Unauthorized ||
+        response.status === HttpStatus.Forbidden
```

Comment:
Same as above: consider a human readable predicate.

Reply:
@ricardovdheijden so a seperate const ? also for the 1st one ?

const isUnauthorized = (status: number) => (status === HttpStatus.Unauthorized || status === HttpStatus.Forbidden)